mitiq/mitiq_qiskit/conversions.py

```diff
@@ -72,4 +89,5 @@ def from_qasm(qasm: QASMType) -> cirq.Circuit:
     Returns:
         Mitiq circuit representation equivalent to the input QASM string.
     """
+    qasm = _remove_barriers(QuantumCircuit.from_qasm_str(qasm)).qasm()
```

Comment:
Do we need to `_remove_barriers` in the body of "from_qasm"? Isn't `_remove_barriers` inside `from_qiskit` enough?